portfolio-project-react/src/components/Portfolio/Portfolio.jsx

```diff
@@ -15,16 +15,21 @@ const Portfolio = () => {
     location.pathname === "/" ? true : false
   );
 
+  const [isDataViewPageRouteEnabled, setIsDataViewPageRouteEnabled] = useState(
+    location.pathname === "dataviewpage" ? true : false
```

Comment:
Using a ternary operator is unnecessary here. Just location.pathname === "dataviewpage" will be enough as a return value.